backend/Dockerfile

```diff
@@ -16,10 +16,14 @@ RUN apk add --no-cache \
 # switch to default user to not chown node_modules on startup
 USER $USER
```

Comment:
Maybe since I'm doing the chown this isn't necessary?

Reply:
Yeah, I also think so. I think we can safely remove all `USER` from this file (this line, 25 and 31).